lua/entities/gmod_tardis/modules/cl_projectedlight.lua

```diff
@@ -79,12 +87,13 @@ ENT:AddHook("Think", "projectedlight", function(self)
 	local shouldoff = self:CallHook("ShouldNotDrawProjectedLight")
 
 	if shouldon and (not shouldoff) then
-        if (not self.projectedlight) or (not IsValid(self.projectedlight)) then
-            self:CreateProjectedLight()
-        else
-            self:UpdateProjectedLight()
-        end
-    elseif self.projectedlight then
-        self:RemoveProjectedLight()
-    end
+		if (not self.projectedlight) or (not IsValid(self.projectedlight)) then
```

Comment:
Super minor but you could just have the UpdatedProjectedLight outside the if statement and only the create one inside it, no else statement